ipn/ipnlocal/local.go

```diff
@@ -870,7 +870,11 @@ func (b *LocalBackend) WhoIs(ipp netip.AddrPort) (n tailcfg.NodeView, u tailcfg.
 	}
 	n, ok = b.peers[nid]
 	if !ok {
-		return zero, u, false
+		// Check if this the self-node, which would not appear in peers.
+		if nid != b.netMap.SelfNode.ID() {
+			return zero, u, false
+		}
+		n = b.netMap.SelfNode
 	}
 	u, ok = b.netMap.UserProfiles[n.User()]
```

Comment:
huh, i guess we should do this for this too :)

Reply:
or just up very early in this method

Reply:
Done.